lib/vector-core/Cargo.toml

```diff
@@ -16,6 +15,7 @@ chrono = { version = "0.4.19", default-features = false, features = ["serde"] }
 db-key = { version = "0.0.5", default-features = false, optional = true }
 derivative = { version = "2.2.0", default-features = false }
 dyn-clone = { version = "1.0.4", default-features = false }
+enrichment = { path = "../enrichment" }
```

Comment:
I could be wrong, but `enrichment` is not actually a dependency of vector-core any longer?

Reply:
It is used, but only when the `vrl` feature is enabled. I'll update.